### PR TITLE
OpenPaas-Suite/esn#25 Fix loading animation

### PR DESCRIPTION
**Pull request by pbson**
Partially fix: OpenPaaS-Suite/esn#25 and OpenPaaS-Suite/esn-frontend-common-libs#86

### CODE REVIEW

File: assets/index.pug [hunk 1]
```diff
@@ -14,13 +14,13 @@ html.loading
...
     header#header.event-consultation
       div.header-inner.flex-vertical-centered.flex-space-between
         ul.list-inline.all-centered
           li.logo
             a(href='#')
               img.openpaas-logo(ng-src="/white-logo.svg", fallback-src="/images/white-logo.svg", src="/white-logo.svg", alt="{{ 'OpenPaas Logo' | translate }}")
-    .consultation-body(ng-view, ng-cloak)
-      include ../src/app/app-loading.pug
\ No newline at end of file
+    .consultation-body(ng-view, ng-cloak )
```
Comment (huy-ta) [inline assets/index.pug:25]:
```suggestion
    .consultation-body(ng-view, ng-cloak)
```

File: src/app/app-loading.pug [hunk 2]
```diff
@@ -1,4 +1,3 @@
-#spash(style='height: 100%; padding: 0; margin: 0; display: flex; align-items:center; justify-content: center; flex-direction: column')
-  div(style='width: 80%; max-width: 150px')
+#spash(style='z-index:100; background:#fff; position:fixed;bottom: 0;right: 0;left: 0;top: 0;height:100%; padding: 0; margin: 0; display: flex; align-items:center; justify-content: center; flex-direction: column')
```
Comment (huy-ta) [inline src/app/app-loading.pug:1]:
```suggestion
#spash(style='z-index:100; background:#fff; position:fixed; bottom: 0; right: 0; left: 0; top: 0; height:100%; padding: 0; margin: 0; display: flex; align-items:center; justify-content: center; flex-direction: column')
```

File: src/app/app-loading.pug [hunk 3]
```diff
@@ -1,4 +1,3 @@
-#spash(style='height: 100%; padding: 0; margin: 0; display: flex; align-items:center; justify-content: center; flex-direction: column')
-  div(style='width: 80%; max-width: 150px')
+#spash(style='z-index:100; background:#fff; position:fixed; bottom: 0; right: 0; left: 0; top: 0; height:100%; padding: 0; margin: 0; display: flex; align-items:center; justify-content: center; flex-direction: column')
```
Comment (huy-ta) [inline src/app/app-loading.pug:1]:
```suggestion
#spash(style='z-index: 100; background: #fff; position: fixed; bottom: 0; right: 0; left: 0; top: 0; height: 100%; padding: 0; margin: 0; display: flex; align-items:center; justify-content: center; flex-direction: column')  
```

:))))

Comment (piavgh) [inline src/app/app-loading.pug:1]: #spash or #splash?

Comment (huy-ta) [reply src/app/app-loading.pug:1]: It's `#spash`, probably a typo from the previous commits that Michael did.

Comment (piavgh) [inline src/app/app-loading.pug:1]: where does this file `app-loading.pug` load? I see that you removed `include ../src/app/app-loading.pug`

Comment (piavgh) [reply src/app/app-loading.pug:1]: I mean the correct id is #splash, right? We should update it

Comment (huy-ta) [reply src/app/app-loading.pug:1]: The correct id is `#spash` (as in it is defined that way in the CSS files), while the correct name should be `splash`. It should be handled in another issue.

Comment (pbson) [reply src/app/app-loading.pug:1]: @piavgh It is for `#wrap(ng-include='session.template')`. If this file is removed, the default loading animation from common libs will be used instead

**Review by huy-ta — APPROVED** [review]

**Review by piavgh — APPROVED** [review]

**Review by tuanlc — APPROVED** [review]

**Pull request merged**